roles/smartgateway/tasks/main.yml

```diff
@@ -4,15 +4,26 @@
   set_fact:
     api_groups: "{{ lookup('k8s', cluster_info='api_groups') }}"
 
+- name: Print some debug information
```

Comment:
Copy pattern from STO for being able to override a single parameter in a default object map.